CvGameCoreDLLUtil/include/CvEnums.h

```diff
@@ -3775,3 +3775,12 @@ enum CLOSED_ENUM RenewalReason
 	REASON_BETTER_RENEWAL_CHOICE = 4,
 };
 #endif
+
+enum CLOSED_ENUM RoutePurpose
```

Comment:
What does actually mean CLOSED_ENUM?

Reply:
As far as I've been able to work out, it means that all values are predefined, as opposed to OPEN_ENUMs which are populated later.

Example OPEN_ENUM:
enum OPEN_ENUM BuildTypes
{
	NO_BUILD = -1,
};

Example CLOSED_ENUM:
enum CLOSED_ENUM RouteTypes
{
	NO_ROUTE = -1,

	ROUTE_ROAD,
	ROUTE_RAILROAD,
	ROUTE_ANY,

	NUM_ROUTE_TYPES ENUM_META_VALUE
};

Reply:
Thx